core/src/main/scala/org/broadinstitute/dsde/rawls/entities/compact/batch/BatchHandling.scala

```diff
@@ -170,25 +170,33 @@ trait BatchHandling extends LazyLogging with AttributeSupport {
   }
 
   /** write this batch of Entity to the database */
-  private def insertBatch(batch: Seq[Entity], allowUpsert: Boolean): ReadWriteAction[Int] =
+  private def insertOrUpdateBatch(batch: Seq[Entity]): ReadWriteAction[Int] =
 
     for {
       // Batch insert to ENTITY table. Save the whole batch first to handle cases where an entity in this batch
       // has a reference to another entity in the same batch.
-      entitiesCreated <- repository.queries.batchCreateEntities(workspaceId, batch, allowUpsert = allowUpsert)
+      entitiesCreated <- repository.queries.batchCreateEntities(workspaceId, batch, insertOnly = false)
```

Comment:
batchInsert and batchUpdate both _always_ allow updating, so `insertOnly` is always false.